gym_xiangqi/test/piece_test.py

```diff
@@ -0,0 +1,24 @@
+import unittest
+from gym_xiangqi.piece import Piece, General, Advisor, Elephant
+from gym_xiangqi.piece import Horse, Chariot, Cannon, Soldier
+
+
+class TestPieceClasses(unittest.TestCase):
+
+    def setUp(self) -> None:
+        self.classes = [Piece, General, Advisor, Elephant,
+                        Horse, Chariot, Cannon, Soldier]
+
+    def test_piece(self):
```

Comment:
Be more descriptive in the test method. Maybe something like `test_piece_initialization`?

Reply:
change made based on this